packages/component/src/SendBox/TypingIndicator.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+
+import { localize } from '../Localization/Localize';
+import connectToWebChat from '../connectToWebChat';
+import TypingAnimation from './Assets/TypingAnimation';
+
+const TypingIndicator = ({ language, lastTypingAt, styleSet }) => {
+  const {
```

Comment:
Destruct `typingIndicator` and `typingAnimationDuration` using line 8 instead of line 8 and 9-11?